Office 365/o365/ingest/parser.yml

```
@@ -162,12 +168,71 @@ stages:
           event.action: event.type
         fallback: '["info"]'
       - set:
+          user.id: "{{json_event.message.MailboxOwnerSid}}"
+          office365.exchange.mailbox_guid: "{{json_event.message.MailboxGuid}}"
+          office365.exchange.modified_properties: "{{json_event.message.ModifiedProperties}}"
           office365.context.aad_session_id: "{{json_event.message.SessionId}}"
 
   parse_exchange_item_group:
     actions:
+      - set:
+          event.category: ["email", "file"]
+
+      - translate:
+        dictionary:
+          "Create": ["info", "creation"]
+          "Update": ["info", "change"]
+          "HardDelete": ["info", "deletion"]
+          "SoftDelete": ["info", "deletion"]
+          "MoveToDeletedItems": ["info", "deletion"]
+          "MailItemsAccessed": ["info"]
+          "MailboxLogin": ["start"]
+        mapping:
+          event.action: event.type
+        fallback: ["info"]
```

According to my previous suggestion:
```suggestion
          event.type: ["info"]
          
      - set:
          event.category: ["authentication"]
          event.type: ["start"]
        filter: "{{final.event.action == 'MailboxLogin'}}"
```